float64[] position
float64[] stiffness
int32[] cube_id
string command
---
float64[] position
int32[] cube_id
float64[] timestamp
